assets/src/edit-story/components/panels/backgroundDisplay/dialogContent.js

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+import { rgba } from 'polished';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { __experimentalCreateInterpolateElement as createInterpolateElement } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { useConfig } from '../../../app/config';
+
+const DialogSection = styled.div`
+  padding: 0px 16px;
+  min-width: 260px;
+
+  h3 {
+    margin: 0px;
+    font-family: ${({ theme }) => theme.fonts.body1.family};
+    font-size: ${({ theme }) => theme.fonts.body1.size};
+    line-height: ${({ theme }) => theme.fonts.body1.lineHeight};
+    letter-spacing: ${({ theme }) => theme.fonts.body1.letterSpacing};
+    font-weight: bold;
+    color: ${({ theme }) => rgba(theme.colors.fg.v0, 0.68)};
+  }
+
+  b {
+    font-weight: bold;
+  }
+
+  p {
+    font-family: ${({ theme }) => theme.fonts.body1.family};
+    font-size: ${({ theme }) => theme.fonts.body1.size};
+    line-height: ${({ theme }) => theme.fonts.body1.lineHeight};
+    letter-spacing: ${({ theme }) => theme.fonts.body1.letterSpacing};
+  }
+
+  mark {
+    background-color: ${({ theme }) => theme.colors.textHighlight};
+  }
+`;
+
+const DialogSeparator = styled.div`
+  margin-right: 42px;
+`;
+
+const Row = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin-bottom: 16px;
+  justify-content: space-between;
+  flex: 1;
+`;
+
+function BackgroundDisplayDialogContent() {
+  const { pluginDir } = useConfig();
+
+  return (
+    <>
+      <Row>
+        {__(
+          'Your story is created at one aspect ratio (1080 x 1920). When published, users will view your stories using phones with aspect ratios that may differ widely.',
```

Comment:
I'd recommend using placeholders for the dimensions so they won't be translated by mistake, and translations won't get invalidated because the dimensions (and thus the whole string) change.

Something like that:

```js
sprintf(
  /* translators: %s: aspect ratio */
  __('Your story is created at one aspect ratio (1080 x 1920). [...]', 'web-stories')
'1080 x 1920'
)
```

Noting that "1080x1920" are the dimensions, not really the aspect ratio (which would be "9/16").

Reply:
another nit here: Can we make sure we use the constants we have set up for these, and not hardcode the numbers? This is especially relevant as there's a change coming imminently (cc @dvoytenko).

Reply:
Absolutely, please update to constants ASAP as we're about to change the ratios.